app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -9,13 +9,12 @@ import androidx.navigation.ui.NavigationUI
 import com.google.android.material.bottomnavigation.BottomNavigationView
 import edu.uc.hickmadc.customnotifications.ui.main.MyListAdapter
 
-//import edu.uc.hickmadc.customnotifications.ui.main.MainFragment
 
 class MainActivity : AppCompatActivity() {
 
     //temp data
-    val language = arrayOf<String>("Brush Teeth","Eat Breakfast","Do schoolwork")
-    val description = arrayOf<String>(
+    private val language = arrayOf<String>("Brush Teeth","Eat Breakfast","Do schoolwork")
+    private val description = arrayOf<String>(
         "Brush teeth for 2 minutes",
         "Eat so you don't feel like shit",
         "")
```

Comment:
I added private behind both lines so they can't be changed outside of this class.

Reply:
Source: https://kotlinlang.org/docs/visibility-modifiers.html#classes-and-interfaces